lib/BloqadeSchema/src/execute.jl

```diff
@@ -3,25 +3,29 @@
 
 
 
-# """
-#     execute(j::String)
-
-# Executes a task given as a JSON string in the task specification API format, and returns a JSON string of the result
-# """
-# function execute(j::String)
-#     task = Configurations.from_dict(BloqadeSchema.TaskSpecification, JSON.parse(j))
-#     h = from_json(j)
-#     return JSON.json(Configurations.to_dict(execute(h, length(content(h[1]).atoms), 3e-6, task.nshots)))
-# end
-
-# function execute(h::Add, n_atoms::Int, total_time::Float64, nshots::Int)
-#     # Always start off with the zero state
-#     reg = zero_state(n_atoms)
-#     problem = SchrodingerProblem(reg, total_time, h)
-#     emulate!(problem)
-#     bitstrings = reg |> measure(; nshots=nshots)
-#     return to_task_output(bitstrings)
-# end
+"""
+    execute(j::String)
+
+Executes a task given as a JSON string in the task specification API format, and returns a JSON string of the result
+"""
+
+function execute(j::String)
+    task = Configurations.from_dict(BloqadeSchema.TaskSpecification, JSON.parse(j))
```

Comment:
One question I have is whether or not this function will give a clear error in the event of a schema json that isn't formatted correctly.

Reply:
what do you mean by formatted correctly?

Reply:
I see, I believe that will cause `from_dict` throwing an error? since fields may in place based on the schema specification defined by `TaskSpecification`